modules/polaris/src/main/ts/ephox/polaris/api/Regexes.ts

```diff
@@ -37,7 +37,7 @@
   (?:\.[A-Za-z0-9-]+)*
 )
 (?::[0-9]+)?
-(?:\/[-+~=%.()\/\w]*)?
+(?:\/[-+~=%.,()\/\w]*)?
```

Comment:
We should mention about this modification in the section above that lists changes from the original.

- [ ] Update modifications section above